src/main/resources/openapi.json

```
@@ -1,314 +1,1279 @@
+                "operationId": "poll",
+                "responses": {
+                    "200": {
+                        "description": "Poll request executed successfully.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/ConsumerRecordList"
+                                },
+                                "examples": {
+                                    "response": {
+                                        "value": [
+                                            {
+                                                "topic": "topic",
+                                                "key": "key1",
+                                                "value": {
+                                                    "foo": "bar"
+                                                },
+                                                "partition": 0,
+                                                "offset": 2
+                                            },
+                                            {
+                                                "topic": "topic",
+                                                "key": "key2",
+                                                "value": [
+                                                    "foo2",
+                                                    "bar2"
+                                                ],
+                                                "partition": 1,
+                                                "offset": 3
+                                            }
+                                        ]
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "404": {
+                        "description": "The specified consumer instance was not found.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Error"
+                                },
+                                "examples": {
+                                    "response": {
+                                        "value": {
+                                            "error_code": 404,
+                                            "message": "The specified consumer instance was not found."
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "406": {
+                        "description": "The format of the consumer does not match the embedded format requested in the Accept header.",
```

It's not immediately obvious what the "format of the consumer" means, and in particular that it was defined by a previous request. Maybe this is a little clearer:
```suggestion
                        "description": "The `format` used in the consumer creation request does not match the embedded format in the Accept header of this request.",
```